app/helpers/signs_helper.rb

```diff
@@ -20,6 +20,10 @@ def render_grammar_notes(sign)
     end.compact.join(', ').html_safe
   end
 
+  def convert_to_high_resolution(sign_drawing)
+    sign_drawing.gsub(/default.png$/, 'high_resolution.png')
```

Comment:
ooh nice change!

Reply:
FYI this logic won't work with edge cases eg `something-Default.png` or `something-default.PNG`. `sign_drawing.gsub(/default.png$/i, 'high_resolution.png')` is what you need. Consider this as 🐘 if case sensitivity is irrelevant.